src/Components/Settings/Settings.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import style from './Settings.module.scss'
+import { SettingsContext } from '../../Context/SettingsContext'
+
+const Settings = () => {
+    const [ settings, setSettings ] = useContext(SettingsContext)
+    
+    const onChange = (event) => {
+        setSettings({ 
+            ...settings, 
+            [event.target.id]: +event.target.value
+        })
+    }
+
+    return (
+        <div className={style.settingsContainer}>
+            <h1 className={style.settingsHeader}>Settings</h1>
```

Comment:
Do we wanna norm on making these subheadings `<h2>` so that we aren't duplicating the `<h1>` in the header?

Reply:
Great call, we should only ever have one H1 on a view, which will be our app name!

Reply:
Great catch!